README.md

```diff
@@ -29,6 +29,62 @@ juju relate ranger-k8s:db postgresql-k8s:database
 ```
 Refer to [CONTRIBUTING.md](./CONTRIBUTING.md) for details on building the charm and ranger image. 
 
+### Group management with Apache Ranger
+The Charmed Ranger Operator makes use of the [Ranger API](https://ranger.apache.org/apidocs/index.html) and [apache-ranger PyPi package](https://pypi.org/project/apache-ranger/) to manage users and groups. The source of users and group memberships is a `user-group-configuration.yaml` file provided to the charm as a configuration value `user-group-configuration`. 
+
+An example of this file is here: 
+```
+ranger-k8s:
+   user-group-configuration: |
+      relation_2:
+         users:
+            - name: user1
+              firstname: One
+              lastname: User
+              email: user1@canonical.com
+            - name: user2
+              firstname: Two
+              lastname: User
+              email: user2@canonical.com
+
+         groups:
+            - name: developers
+              description: users with developer level access.
+            - name: users
+              description: users with select only access.
+
+         memberships:
+            - groupname: users
+              users: [user1, user2]
+            - groupname: developers
+              users: [user2]
+```
+The charm will automatically sync users and groups from the configuration file to Ranger admin. Removing groups and group memberships were required.
```

Comment:
Sorry, this was a typo, it should say `Removing groups and group memberships where required`, which still may not be clear but it is essentially saying that as well as creating new groups it will also delete old ones that are no longer present in the file. So don't add groups directly in the interface, only do it via this file. I will re-word the readme. Let me know if this still does not make sense please!

Reply:
I've updated the readme to be more clear.